arch/arm/core/aarch64/arm_mmu.c

```diff
@@ -26,75 +27,88 @@
 #define MMU_DEBUG(...)
 #endif
 
-/* We support only 4kB translation granule */
+#if DUMP_PTE
```

Comment:
Not sure I get this @carlocaione , DUMP_PTE is defined above as 0; does that activate the following #if block?

Reply:
That's not my code (I only moved the debug stuff all together at the start of the file) but you are definitely right. Isn't that true for `MMU_DEBUG_PRINTS` as well?

Reply:
That's not my code but `DUMP_PTE` is 0 so `if 0` is making the following block to be excluded.

Reply:
@ioannisg `DUMP_PTE` macro is made available for debug purpose. When enabled, it will print *all* the page table entries as are being created. `MMU_DEBUG_PRINTS` need to enabled for basic debug, which just initializes MMU driver before UART/UART console driver- so that we see prints from MMU driver, and `DUMP_PTE` needs to enabled on top for advanced PTE level debug. I hope this clarifies.